### DISCUSSION TITLE
How to use select component with useFieldArray

### DISCUSSION THREAD

**Discussion by dinhmai74** [react-hook-form/react-hook-form#4069] [Q&A]
I got an error when using Select component with Controller ( by render props ). It'll append an undefined value. Did I use it in the wrong way? @bluebill1049 
![image](https://user-images.githubusercontent.com/40544043/106413679-479b9400-647d-11eb-9e7a-09718bb04258.png)

https://codesandbox.io/s/react-hook-form-usefieldarray-forked-zfdvj?file=/src/index.js:1260-1307

**Comment by bluebill1049** [comment]
<img width="414" alt="Screen Shot 2021-02-01 at 3 17 33 pm" src="https://user-images.githubusercontent.com/10513364/106414395-9d356800-64a0-11eb-864d-a74db9b7cf6e.png">
you are missing a `defaultValue` prop

**Reply by dinhmai74** [reply]
Still not working, the default value in form still is undefined @bluebill1049 

![image](https://user-images.githubusercontent.com/40544043/106417075-b2e96400-6485-11eb-97c2-5243d65fd9e9.png)

**Reply by bluebill1049** [reply]
`defaultValue` at Controller. 

take a look at the doc: https://react-hook-form.com/api#useFieldArray there is a codesandbox example that may help.
